### PR TITLE
Add `runtests.jl` and github workflow for CI

### PR DESCRIPTION
**Pull request by ghislainb**
This is a first step to setting up the CI of the package. In this PR, only the "Helmholtz" tutorial is tested.
Once the CI is operational, I will try to trigger it automatically from "Bcube.jl" for downstream integration tests.

Then, if we want a more robust integration test process, we will have to include the other tutorials and examples in new `testsets` in `runtests.jl` (with new PRs).

### CODE REVIEW

File: src/tutorial/helmholtz.jl [hunk 1]
```diff
@@ -45,31 +45,14 @@ println("Running Helmholtz example...") #hide
...
-
-vp, vecp = eigen(Matrix(A), Matrix(B))
-@show sqrt.(abs.(vp[3:8]))
+# @__PLAIN_CODE__
 
 # # Commented code
 # Load the necessary packages
 using Bcube
 using LinearAlgebra
+using Test #src
```
Comment (bmxam) [inline src/tutorial/helmholtz.jl:55]: maybe we can move this `using Test` inside the `if get(ENV...`?

File: src/tutorial/helmholtz.jl [hunk 2]
```diff
@@ -107,37 +90,42 @@ vp, vecp = eigen(Matrix(A), Matrix(B))
...
+    ref_results = [                                         #src
+        3.144823462554393,                                  #src
+        4.447451992013584,                                  #src
+        6.309054755690625,                                  #src
+        6.309054755690786,                                  #src
+        7.049403274103087,                                  #src
+        7.049403274103147,                                  #src
+    ]                                                       #src
+    @test all(results .≈ ref_results)                       #src
+    files2check = outputvtk .* [".pvd", "_00000000.vtu"]    #src
```
Comment (bmxam) [inline src/tutorial/helmholtz.jl:127]: if it's to show how we could compare the results using binary files ok. But in this precise case (helmholtz), I think the comparison with the ref results is enough (and better). In my opinion, checking the file hash should be avoid when possible : if `WriteVTK` decide to add one comment in the file, all these tests will fail.

**Review by bmxam — APPROVED** [review]

File: src/tutorial/helmholtz.jl (hunk 1 above)
Comment (ghislainb) [reply src/tutorial/helmholtz.jl:55]: I already tried and it does not work... I don't know why

File: src/tutorial/helmholtz.jl (hunk 2 above)
Comment (ghislainb) [reply src/tutorial/helmholtz.jl:127]: I agree and feel free to correct it in a future PR. And yes, for now I am only focusing on setting up the whole process (with file checking and so on)

Comment (ghislainb) [reply src/tutorial/helmholtz.jl]: Done !

**Review by bmxam — APPROVED** [review]

**Pull request merged**